### DISCUSSION TITLE
article documents guid changes

### DISCUSSION THREAD

**Discussion by rogerg1967** [openstore-ecommerce/OpenStore#131] [General]
Hi,
I have an issue where we have several documents connected to a single article. It has been working fine until we upgraded the dnn core from dnn 9.1.1 to 9.8.1.
We are finding now that the first document in the db xml field the guid is being replaced by one of the others associated with the same article.
I have attached the xml to show the issue if you look at line 6 and line 63 you will see the issue. This now means the wrong pdf is being downloaded when the first article link is selected.
Can you advise what may be causing this issue?
Roger

[buzzard xml issue.txt](https://github.com/openstore-ecommerce/OpenStore/files/6090616/buzzard.xml.issue.txt)

**Comment by leedavi** [comment]
Hi Roger,

I see, yes it's wrong.  I'm unsure why.  We have OpenStore running on DNN9.8 and have not noticed this.  But I don;t think we have more that one document on that system.

So, I have done a quick test on my dev system (v4.1.5) and I cannot recreate.

What version of OpenStore is it?  And did you upgrade that also?

Can you recreate the problem?

**Comment by rogerg1967** [comment]
Its quite an old version 3.6.7. There is a lot of customisation, do you think it would upgrade?

**Reply by leedavi** [reply]
The DB is nearly the same and I have updated 3 or 4 sites.   

https://github.com/openstore-ecommerce/OpenStore/blob/master/Installation/Readme_NBS_Upgrade.txt

However, if you have bespoke modules and plugins you should leave it in v3.    The problem you have is that v3 was never designed for DNN9.  Leaving it in DNN8 would probably have been OK for the next 10 years,

Having said that, if your changes are only design and templates, then an upgrade could be possible, but you'll need to alter the theme your using to match your changes.

The only time I fail to upgrade was a client that had bespoke modules and plugins.   I could have got it all to work, but the cost for me to update all the modules was simply too much, so it continues to run in DNN8 and v3.

**Comment by KathyBar** [comment]
Well, these documents and articles don't help at all. I've tried those. You'd better try to [buy an argumentative essay](https://perfectessaysonline.com/buy-argumentative-essay) that will be more useful than just looking through the articles.

**Reply by DNNMonster** [reply]
It's open source remember.  Hopefully we get things right but we're human.  Dave is one of the most helpful open source authors on github from my experience.  If you can share more details about your issue then we can help you get to your goal.  Multi cultural documentation is challenging but the upgrade is doable.  You're best with any major upgrade to do a full back up and try performing the upgrade in an environment outside of production.  You'll be able to identify the obstacles your installation has to overcome before doing the same on your production site.  You can also help us improve the documentation if you're willing to share some of your time.

**Reply by leedavi** [reply]
Hi Kathy,
Sorry you feel that way, I do understand your frustration when documentation does not help.  Which documentation and articles do you refer to?
If you mean the GitHub pages, then they are not the documentation and we have only just moved the forum to here, to try and save support time and costs, which apply with an independent forum.

You can try these documents : https://doc.openstore-ecommerce.com/

There are also developer documents: https://doc.openstore-ecommerce.com/Developer-Guide

The manual for operation is only in French, we simply have not had time to translate to English, even though it is prepared: https://manual.openstore-ecommerce.com/fr-fr/

But, we as still along way behind where we want to be with documentation.

If you have any contribution to documentation, we'll be very happy to include it.
